pkg/nimble/autoadv/nimble_autoadv.c

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_nimble_autoadv
+ *
+ * @{
+ *
+ * @file
+ *
+ * @author      Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <errno.h>
+#include <stdlib.h>
+
+#include "nimble_riot.h"
+
+#include "host/ble_hs.h"
+#include "host/ble_gap.h"
+#include "net/bluetil/ad.h"
+
+#include "nimble_autoadv.h"
+
+static struct ble_gap_adv_params _advp;
+
+static int32_t _adv_duration;
+
+static uint8_t buf[BLE_HS_ADV_MAX_SZ];
+static bluetil_ad_t _ad;
```

Comment:
Can you add some comments for these variables?

Reply:
I added some comments.